app/controllers/first_draft_collections_controller.rb

```diff
@@ -25,9 +25,12 @@ def create
     @form = collection_form(collection_version)
 
     if @form.validate(create_params) && @form.save
-      collection_version.collection.event_context = { user: current_user }
+      collection_version.collection.event_context = { user: current_user, description: 'Created' }
       collection_version.update_metadata!
-      collection_version.begin_deposit! if deposit_button_pushed?
+      if deposit_button_pushed?
+        collection_version.collection.event_context = { user: current_user }
+        collection_version.begin_deposit!
+      end
```

Comment:
this ensures we don't also say "Created" for the "Deposited" event description that this will generate